packages/zwave-js/src/lib/driver/Driver.ts

```diff
@@ -1252,6 +1252,9 @@ export class Driver extends EventEmitter {
 		this._wasDestroyed = true;
 		this.driverLog.print("destroying driver instance...");
 
+		// destroy loggers
+		this.logContainer.destroy();
```

Comment:
that comes too early, we're using the loggers below.

Reply:
arggg sorry you right